config.ru

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
```

Comment:
redudant?

Reply:
It's needed (and encouraged by rubocop) - it's to make strings immutable.

https://stackoverflow.com/questions/37799296/ruby-what-does-the-comment-frozen-string-literal-true-do